Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -118,7 +120,8 @@ function commitRelease(apiServer: string, apiVersion: string, appSlug: string, u
     tl.debug(`---- url: ${commitReleaseUrl}`);
     let headers = {
         "X-API-Token": token,
-        "User-Agent": userAgent
+        "User-Agent": userAgent,
+        "internal-request-source": "VSTS"
```

Comment:
Is this help to mobile center indicate the upload is happening from VSTS? The "userAgent" string also contains "VSTS" but they will need to parse it.